### PR TITLE
groups renamed into roles

### PR DESCRIPTION
**Pull request by fvanderbiest**
See https://github.com/georchestra/georchestra/issues/1415

### CODE REVIEW

File: build_support/shared.maven.filters [hunk 1]
```diff
@@ -80,13 +80,13 @@ shared.ldapUrl=ldap://@shared.ldap.host@:@shared.ldap.port@
...
 # shared.ldap.userSearchBaseDN is the query filter for users
-# shared.ldap.groupSearchBaseDN is the query filter for groups
+# shared.ldap.roleSearchBaseDN is the query filter for roles
 # shared.ldap.admin.dn is the query filter for the ldap admin user
 # shared.ldap.admin.password is the ldap admin user password
 
 shared.ldap.baseDn=dc=georchestra,dc=org
 shared.ldap.userSearchBaseDN=ou=users
-shared.ldap.groupSearchBaseDN=ou=groups
+shared.ldap.roleSearchBaseDN=ou=roles
```
Comment (fvanderbiest) [inline build_support/shared.maven.filters]: Well I'm not sure we should change groupSearchBaseDN into roleSearchBaseDN for now ...

**Comment by Vampouille** [comment]
Looks good to me !

**Comment by Vampouille** [comment]
I think it's a good idea to change key in configuration like groupSearchBaseDN into roleSearchBaseDN. But we also need to change class name ... so this will introduce many changes.

**Comment by fvanderbiest** [comment]
> so this will introduce many changes

I agree with you. 
We'll do it later, when all LDAP related changes are over.

**Pull request merged**